bluebottle/members/tests/test_admin.py

```diff
@@ -301,13 +301,15 @@ class MemberPlatformSettingsAdminTestCase(BluebottleAdminTestCase):
 
     def test_require_location(self):
         self.app.set_user(self.superuser)
+        user = BlueBottleUserFactory.create()
         page = self.app.get(reverse('admin:members_memberplatformsettings_change'))
         form = page.forms[0]
         form['require_office'].checked = True
 
         form.submit()
         settings_platform = MemberPlatformSettings.load()
-        self.assertTrue(settings_platform.require_office)
+        if user.place is None:
```

Comment:
How did you verify that this test actually tests anything? Always make sure that you see a newly written test fail. Either by writing the test, letting it fail and writing the code that fixes the test. Or, if you write a test for something that is already working, by termporarily breaking the code. Then run the test, see it fail, unbreak the code, run the test and see it pass.

There is no way that this test would ever break on the assertion you added.